pkg/sync/http/http_sync.go

```diff
@@ -143,10 +142,9 @@ func (hs *Sync) fetchBodyFromURL(ctx context.Context, url string) ([]byte, error
 }
 
 func (hs *Sync) generateSha(body []byte) string {
-	hasher := sha1.New() //nolint:gosec
+	hasher := sha3.New256() //nolint:gosec
```

Comment:
```suggestion
	hasher := sha3.New256()
```

Do you know if this nolint is still needed?

Reply:
not sure, but I would guess no